src/test/java/org/opensearch/security/dlic/rest/api/SavedTenantApiTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package org.opensearch.security.dlic.rest.api;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.opensearch.security.securityconf.impl.CType;
+import org.apache.http.Header;
+import org.apache.http.HttpStatus;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.xcontent.XContentType;
+import org.junit.Assert;
+import org.junit.Test;
+import org.opensearch.security.test.helper.rest.RestHelper.HttpResponse;
+import java.util.Arrays;
+
+import static org.junit.Assert.*;
+
+@RunWith(Parameterized.class)
+public class SavedTenantApiTest extends AbstractRestApiUnitTest {
+    private final String BASE_ENDPOINT;
+    private final String ENDPOINT;
+
+    public SavedTenantApiTest(String baseEndpoint, String endpoint){
+        BASE_ENDPOINT = baseEndpoint;
+        ENDPOINT = endpoint;
+    }
+
+    @Parameterized.Parameters
+    public static Iterable<String[]> endpoints() {
+        return Arrays.asList(new String[][] {
+                {"/_opendistro/_security/api/", "/_opendistro/_security/api/account"},
+                {"/_plugins/_security/api/", "/_plugins/_security/api/account"}
+        });
+    }
+
+    @Test
+    public void testDefaultTenantValueCheck() throws Exception {
+        // arrange
+        setup();
+        final String testUser = "test-user";
+        final String testPass = "test-pass";
```

Comment:
Variable name of constants needs to be all capital letters with _ separators like BASE_ENDPOINT